docs/LearnCalico.md

```diff
@@ -0,0 +1,358 @@
+# Calico networking in a containerized environment
+
+This GitHub repository is the entry point for running a Calico network in a
+containerized environment.
+
+The `calico-docker` repository allows users to easily configure endpoints and 
+network policy with the `calicoctl` command line tool. `calico-docker` utilizes 
+the Calico Felix process from within a container to interface with a host's 
+Linux Kernel, providing configurable connectivity between containers in a 
+network and the outside world.
+
+
+The repository provides:
+
+- A `calico/node` docker image that is used to run the Calico Felix agent 
+  (responsible for IP routing and ACL programming) and a BGP agent (BIRD) used
+  for distributing routes between hosts.
+- A command line tool `calicoctl` used for starting a `calico/node` container,
```

Comment:
Nitpicky, but instead of starting these bullets with "A", start them with "The"?